### ISSUE TITLE
LiveReport Page Export: When page contains table that does not fit on one page, exporter adds empty page at beginning

### ISSUE THREAD

**Issue by kschmiechen** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#145]
### Current Behavior

when exporting a livereport page that contains a table at the beginning of the document that does not fit on one pdf page, the exported pdf has extra empty page at the beginning (after cover sheet)

### Expected Behavior

exported pdf does not have additional empty page before content

### Steps To Reproduce

1. Create live report page
2. add pdf export button in top region
3. in below region inside a script block widget, add html table with enough rows that the table does not fit on one pdf page
4. export pdf

### Environment - OS

Windows

### Polarion version

2404

### Extension Version

6.0.0

**Comment by grigoriev** [comment]
Hi @kschmiechen,

unfortunately I can not reproduce the problem.

here is my test LiveReport:
<img width="1306" alt="image" src="https://github.com/user-attachments/assets/25e645a3-5bce-4323-82f6-790547004999">
Export button in top region and HTML table generated in script block widget using this Velocity code:

```velocity
#set($rows = 100)  ## Define the number of rows
#set($columns = 10)  ## Define the number of columns

<html>
<head>
    <title>Long HTML Table</title>
    <style>
        table {
            width: 100%;
            border-collapse: collapse;
        }
        table, th, td {
            border: 1px solid black;
        }
        th, td {
            padding: 8px;
            text-align: center;
        }
        th {
            background-color: #f2f2f2;
        }
    </style>
</head>
<body>
    <h2>Long HTML Table Generated by Velocity</h2>
    <table>
        <tr>
            #foreach($column in [1..$columns])
                <th>Column $column</th>
            #end
        </tr>
        #foreach($row in [1..$rows])
            <tr>
                #foreach($column in [1..$columns])
                    <td>Row $row, Col $column</td>
                #end
            </tr>
        #end
    </table>
</body>
</html>
```

The resulting PDF does not contain any empty page:
<img width="527" alt="image" src="https://github.com/user-attachments/assets/cf61b361-efd9-4ff3-93d2-7d4fa5f40bdd">


Could you please provide me with more details how to reproduce the problem?

**Comment by kschmiechen** [comment]
@grigoriev 
When I do this, I get the following result:
[Playground Reports sbb live report export 28871.pdf](https://github.com/user-attachments/files/16560541/Playground.Reports.sbb.live.report.export.28871.pdf)
I completely reset the configuration settings of the exporter to make sure I did not break anything in the CSS.

**Comment by grigoriev** [comment]
@kschmiechen,
thank you, reproduced with the latest CSS.

Will be investigated further. Temporary fix is comment these lines in CSS:

    tr { /* Fixes breaking of work item custom field tables */
        /* page-break-inside: avoid; */
        /* break-inside: avoid; */
    }

**Comment by grigoriev** [comment]
fixed in CSS provided in v6.0.1

**Comment by kschmiechen** [comment]
confirmed fixed
